app/components/Views/Settings/NetworksSettings/NetworkSettings/CustomNetworkView/CustomNetwork.test.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { backgroundState } from '../../../../../../util/test/initial-root-state';
+import renderWithProvider from '../../../../../../util/test/renderWithProvider';
+import CustomNetwork from './CustomNetwork';
+import { CustomNetworkProps } from './CustomNetwork.types';
+import { PopularList } from '../../../../../../util/networks/customNetworks';
+
+const getMockState = () => ({ engine: { backgroundState } });
+
+jest.mock('@react-navigation/native', () => {
+  const actualNav = jest.requireActual('@react-navigation/native');
+  return {
+    ...actualNav,
+    useNavigation: () => ({
+      navigate: jest.fn(),
+    }),
+  };
+});
+
+describe('CustomNetwork component', () => {
+  const getMockCustomNetworkProps = () => {
+    const mockCloseNetworkModal = jest.fn();
+    const mockShowNetworkModal = jest.fn();
+    const mockCustomNetworkProps: CustomNetworkProps = {
+      showPopularNetworkModal: false,
+      isNetworkModalVisible: false,
+      closeNetworkModal: mockCloseNetworkModal,
+      selectedNetwork: {
+        chainId: '0x1',
+        nickname: 'Ethereum Mainnet',
+        rpcUrl: 'https://mainnet.infura.io/v3/your-api-key',
+        ticker: 'ETH',
+        rpcPrefs: {
+          blockExplorerUrl: 'https://etherscan.io',
+        },
+      },
+      showNetworkModal: mockShowNetworkModal,
+      shouldNetworkSwitchPopToWallet: false,
```

Comment:
S: have showAddedNetworks defined in mock with other props, for readability.
**Suggestion part 1/2**

```suggestion
      shouldNetworkSwitchPopToWallet: false,
      showAddedNetworks: true,
```

Reply:
Fixed in https://github.com/MetaMask/metamask-mobile/pull/15283/commits/3a84fb35a75f6d60978e5fd1651ddfdae77a2898